### ISSUE TITLE
2.3.0-beta1 Release plan

### ISSUE THREAD

**Issue by trackleft** [az-digital/az_quickstart#1561]
# Minor Release
## 2.3.0-beta1
This is a release candidate for the next minor version (feature release) of Arizona Quickstart 2. Release candidates are not supported for production sites, but they are intended for widespread testing in preparation for the upcoming stable release. 

#### New features
- #1109
- #1286
- #1518
- #1420
- #1468
- #1539

#### Breaking or visual change to existing behavior
- #1450
- #1467
- #1526
- #1400

#### Risky or disruptive cleanup to comply with coding standards
- #1499
- #1556

#### High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- #1349

#### Drupal core minor level update
- #1181

#### Drupal contrib project updates
- #1473
- #1554

#### Other
- #1549
- #1562

**Comment by joeparsons** [comment]
## 2.3.0-rc1 pick list

(commits merged into `main` since we created the `2.3.x` branch)
```
git cherry-pick c7a85be79838349f9ec766fd5cdda0e7d56a60ac
git cherry-pick 5f57c3ea9c353015e2b140305f83596a41b2db79
```

## 2.3.0-rc1 Release Notes
```
## New changes (not included in 2.2.x)

#### New features
- Closes #1103: Add new AZ News view displays (#1539)
- Closes #821 Adding process plugin for datetime to smart_date to fix duration field. (#1468)
- Closes #1500 Feature display mode for the news feature (#1420)
- Closes #1490: Adding content admin/editor permissions for node clone, block classes, metatag, reports (#1518)
- #1280: Add some functional tests to check that modules can be uninstalled and reinstalled without errors. (#1286)
- Closes #880: Add new ViewsReferenceMapping process plugin with views argument lookup handling. (#1109)

#### Breaking or visual change to existing behavior
- Closes #1363 Use navbar_offcanvas_attributes set in .theme file. (#1400)
- Closes #1207 - Change card breakpoints to match Bootstrap 4 (break sooner) (#1526)
- Closes #1462 Optimize javascript behaviors for full width elements. (#1467)
- Closes #1404: Push sidebar down and fix horizontal scroll problem. (#1450)

#### Risky or disruptive cleanup to comply with coding standards
- Closes #1555: Cleanup existing PHPCS warnings ahead of 2.3.0. (#1556)
- Closes #1498: Update eslint and prettier (#1499)

#### High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- Closes #1348: Remove deprecated ParagraphsBehavior migrate process plugin. (#1349)

#### Drupal core minor level update
- #1340, Closes #1180: Prepare Quickstart for Drupal core 9.3.x. (#1181)

#### Drupal contrib project updates
- Update drupal/media_migration requirement from 1.0.0-alpha13 to 1.0.0-alpha14 (#1554)
- Closes #1472: Consolidated contrib module updates for 2.3.x (#1473)

#### Other
- #1561: 2.3.x branch-specific changes for CI configs. (#1565)
- Closes #1548: Clean up existing configuration drift ahead of Drupal 9.3.x / Quickstart 2.3.0. (#1549)

## Backported changes (also included in 2.2.x)

#### Bug fix
- Fixes #1535 Allow subthemes to override paragraphs and alphabetical listing. (#1536)
- Closes #1394: Updating views re-order notice blocks to use render arrays and Url::fromRoute. (#1440)

#### Accessibility, performance, or security improvement
- Closes #1484: Provide default Antibot module config overrides in Quickstart Security module. (#1485)

#### Critical institutional link or brand change
- Update Arizona Bootstrap to 2.0.14 (#1547)

#### Drupal core - Patch level release (non-security bug-fix release)
- Update drupal/core-recommended requirement from 9.2.18 to 9.2.19 (#1553)

#### Drupal contrib projects - Add new module
- Closes #1501: Add environment_indicator module. (#1503)

#### Other
- #1561: Update composer branch alias for dev-main to 2.4.x-dev. (#1562)
- Closes #1299: Adding documentation surrounding developing and applying updates to quickstart sites. (#1300)
- Closes #1505: Enable errorexit for ProboCI migration tests. (#1506)
- Add info about PHPStan to contributing document. (#1383)
```

**Comment by joeparsons** [comment]
Released 2022-05-17
